bounties/level-1/create-bounties.md

```diff
@@ -0,0 +1,51 @@
+# Creating Bounty Suggestions
+
+## Award
+| Hunter | Agent | Type
+| :- | :- | :-
+| 200 XLM per submission | 0 Credits | Continuous
+
+## Context
+https://github.com/tyvdh/stellar-quest-bounties/issues?q=create-bounties.md
+
+## Description
+
+### What is this task?
+
+The quintessence of Stellar Quest Bounties are - *drumroll* - the bounties!  
+To be able to provide interesting but also useful projects every idea is desirable.  
+To do justice to the idea of Stellar Quest Bounties, what else than to host a bounty hunt to create new bounties! _Bounty-ception!_  
+
+### What are the requirements for the bounty hunter?
+This task involves suggesting useful bounties to the Stellar Quest Bounty repository.  
+_Suggested bounties must be related to the Stellar ecosystem._
+
+Following standards are to be incorporated:
+ 
+* The [given template file](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/README.md) must be used as a baseline for all submissions.
+
+* The level and payout should be determined by difficulty / effort of the task given.  
+
+* Typical payouts and difficulty assessments for each level can be found in the related `level-{n}/README.md`
+  - [Level 1](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-1/README.md)
+  - [Level 2](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/README.md)
+  - [Level 3](https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-3/README.md)
+
+* The submitted Markdown file must be placed in the folder of the given difficulty and named after the bounty title. Example given: `level-{n}/my-bounty-name.md`.
+
+Overall there is a lot of freedom regarding possible quests therefor: be creative!  
+
+### What are the deliverables?
+
+#### Description
+  - The developer must provide a PR to the [SQ Bounties Repository](https://github.com/tyvdh/stellar-quest-bounties) adding the Markdown file describing the proposed bounty.
+#### Review Criteria
+  - The bounty is considered to be complete as soon as the PR is approved and merged.
+
+## Links
+
+- https://github.com/tyvdh/stellar-quest-bounties
+
+## Example
+- https://github.com/tyvdh/stellar-quest-bounties/pull/13
+- https://github.com/tyvdh/stellar-quest-bounties/pull/9
```

Comment:
Just for beauty sake let's format this consistently with spacing. Line 41 butts right against 42, there should be a space. 43 has a tab, 47, 50 and 51 do not. There's a space between ##Links and the first link but that pattern isn't used elsewhere. I'm a stickler for clean consistent code.